### PR TITLE
New Prospect Form Script for v1.1

### PR DESCRIPTION
**Pull request by dominicaw**
### 🚧 Changed
Added a new script `offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js`
This script checks the url params for `option` and chooses where to redirect based on the value.

### CODE REVIEW

File: offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    };
+
+    window.$parsleyLoaded(function (window, document, parsley) {
+      const form = $(prospectFormId);
+      const formContainer = $(formContainerClass);
+      const urlParams = new URLSearchParams(window.location.search);
+      let option = '';
+      if (urlParams.has('option')) {
+        option = urlParams.get('option');
+      }
```
Comment (dominicaw) [inline offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js:49]: Check for option param so option variable can be available for both `onSubmitForm` and `startProspectPolling` functions

File: offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                setTimeout(poll, 1000);
+              },
+              200: function (data) {
+                clearTimeout(buttonLabelTimer);
+                trackSubmission();
+                if (option === 'call_booked') {
+                  window.location.href = `https://meetings.hubspot.com/nickreid/sdr-eu-booking`;
+                } else {
+                  window.location.href = `/hire/get-started/account?prospect_id=${prospectId}`;
+                }
```
Comment (dominicaw) [inline offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js:93]: Handle where form redirects based on `option`

File: offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+            disableSubmitButton(dataBusy);
+          }, 5000);
+
+          // Hide errors
+          form.find('.js-missing-fields').hide();
+          const prospectProperties = Object.assign({}, formProperties, {
+            referrer: location.href,
+            report_source: searchParams.get('report_source'),
+            page_variant_meta: `${window.pageVariantMeasureStart}-optimize-meta-${pageVariantMeasureEnd}`,
+            access_option: `${option}`,
```
Comment (dominicaw) [inline offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js:148]: pass through the option selected in the payload

File: offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+  const formContainerClass = '.js-company-prospect-form';
+  // END FORM TARGETING
+  // This needs to run regardless of whether JQuery has loaded
+  // Disable button
+  const formSubmitButton = document.querySelector(
+    `${prospectFormId} ${formSubmitButtonSelector}`
+  );
+
+  // Store button labels for state changes
+  let initialButtonValue = formSubmitButton.getAttribute('data-initial');
```
Comment (jethroflanagan) [inline offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js:15]:
```suggestion
  let initialButtonValue = formSubmitButton.value();
```
Should also work, reduces the need for extra attributes

**Review by jethroflanagan — APPROVED** [review]
Looks good, minor recommendation, but not critical

**Pull request merged**